test cases/unit/99 custom target name/subdir/meson.build

```diff
@@ -0,0 +1,9 @@
+# Name argument is optional and should default to 'file.txt', but should be
+# displayed as 'subdir/file.txt' by ninja.
+custom_target(
+  input: '../file.txt.in',
+  output: ['@BASENAME@'],
+  command: [python, '--version'],
+  build_by_default: true,
+  capture: true,
+)
```

Comment:
trivial nit: missing newline.

Hopefully with the editorconfig improvement in 173a40be330c1aedec9f7f99ebca67a5edd25c3d this sort of thing will never happen again and no one will need to think about the topic again.

For now we might as well fix it. :)

Reply:
fixed.